ci/lint/04_install.sh

```diff
@@ -18,22 +18,15 @@ if [ ! -d "${PYTHON_PATH}/bin" ]; then
   )
   # For dependencies see https://github.com/pyenv/pyenv/wiki#suggested-build-environment
   ${CI_RETRY_EXE} apt-get install -y build-essential libssl-dev zlib1g-dev \
-    libbz2-dev libreadline-dev libsqlite3-dev curl llvm \
+    libbz2-dev libreadline-dev libsqlite3-dev curl clang llvm \
     libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
-  python-build "$(cat "${BASE_ROOT_DIR}/.python-version")" "${PYTHON_PATH}"
+  # Using clang to workaround https://github.com/pyenv/pyenv/issues/2359.
```

Comment:
> How/why does using Clang solve the segmentation fault issue?

I have no answers, unfortunately.

> Also seems unecessary to put this in the second commit, instead of the first.

There were no problems with gcc on bionic.

Reply:
> I have no answers, unfortunately.

Well linking to this thread, that doesn't explain anything, and just has a comment with a link to a comment in a different thread, where someone claimed that using Clang magically solved their problem, is more confusing than useful.

Reply:
Presumably a gcc compiler bug?

Reply:
> Presumably a gcc compiler bug?

Perhaps. But do we really want to dig into gcc bugtracker for the sake of building an old python?

Reply:
We'll need clang either way, because it uses less memory

Reply:
Does this comment look good
```
# Using clang for less memory footprint and to workaround https://github.com/pyenv/pyenv/issues/2359.
```
?

Reply:
> Also seems unecessary to put this in the second commit, instead of the first.

Done.

> I don't think there is any value in linking to [pyenv/pyenv#2359](https://github.com/pyenv/pyenv/issues/2359).

Dropped.